libnetwork/drivers/bridge/errors.go

```diff
@@ -104,17 +104,6 @@ func (uat ErrUnsupportedAddressType) Error() string {
 // InvalidParameter denotes the type of this error
 func (uat ErrUnsupportedAddressType) InvalidParameter() {}
 
-// ActiveEndpointsError is returned when there are
-// still active endpoints in the network being deleted.
-type ActiveEndpointsError string
-
-func (aee ActiveEndpointsError) Error() string {
-	return fmt.Sprintf("network %s has active endpoint", string(aee))
```

Comment:
I thought; this would still be used for other cases, but it looks like we have a whole duplicate set of these errors in libnetwork itself 😞 https://github.com/moby/moby/blob/12c7411b6b7314bef130cd59f1c7384a7db06d0b/libnetwork/error.go#L73-L85